### DISCUSSION TITLE
Adding links to a header

### DISCUSSION THREAD

**Discussion by koshelekapi** [squidfunk/mkdocs-material#4622] [Q&A] [answered]
Hello dear all!

I'm using MkDocs Material for my documentation site. For now, I need to add some permanent links to the header bar, so they should appear on every page.
As far as I understand, I need to override the **header.html** file, but neither Mkdocs nor Jinja documentation didn't help me to write a script allowing to show these links.
Did anybody solve such an issue?

**Comment by squidfunk** [comment]
Possibly related:

- #4612

**Comment by koshelekapi** [comment, marked as answer]
Solved this problem by extending the `announce` block in `base.html`, as described [there](https://github.com/squidfunk/mkdocs-material/discussions/3008)
